include/gatekeeper_launch.h

```diff
@@ -74,6 +74,10 @@ int
 launch_at_stage3(const char *name, lcore_function_t *f, void *arg,
 	unsigned int lcore_id);
 
+/* Get the number of lcores used except for blocks @block_names. */
+uint16_t
```

Comment:
1. Update comment once parameter `block_names` change name.

2. Add the comment: `This function can be called any time before stage 3 begins.`